taskcluster_util/taskcluster_download.py

```diff
@@ -87,9 +86,8 @@ def run(self):
                 task_id = task_finder.get_taskid_by_namespace(task_namespace)
                 logger.info('The TaskID of Namespace [{}] is [{}].'.format(task_namespace, task_id))
             except TaskclusterRestFailure as e:
-                logger.warning('Can not get the TaskID due to [{}]'.format(e.message))
                 logger.error(e.body)
```

Comment:
After taking a closer look, `e.body` if defined in TaskCluster errors only. I understand why we log the values here. I'm thinking of a way to simplify `run()` a little bit more. What do you think, if we remove every try/except in `run()`, and change main() this way?
```
def main():
    try:
        DownloadRunner().run()
    except Exception as e:
        logger.error(e)
        if e.__dict__:
            logger.error(e.__dict__)
        exit(1)
```

Printing the __dict__ will enable us to output all the details of the exception, how ever the exception is made.